dotCMS/src/integration-test/java/com/dotcms/concurrent/scheduler/DotSchedulerImplTest.java

```diff
@@ -0,0 +1,91 @@
+package com.dotcms.concurrent.scheduler;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.concurrent.DotConcurrentFactory;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UUIDGenerator;
+import com.github.kagkarlsson.scheduler.Scheduler;
+import com.github.kagkarlsson.scheduler.task.helper.OneTimeTask;
+import com.github.kagkarlsson.scheduler.task.helper.Tasks;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Random;
+
+public class DotSchedulerImplTest {
+
+    private final DotScheduler dotScheduler = DotConcurrentFactory.getInstance().getScheduler();
+
+    private final OneTimeTask<DotTask> oneTime = Tasks
+            .oneTime(
+                    "one-time-task",
+                    DotTask.class)
+            .execute(dotScheduler::executeOnce);
+
+    @CloseDBIfOpened
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)